src/emails/components/BreachCard.tsx

```diff
@@ -0,0 +1,93 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ExtendedReactLocalization } from "../../app/functions/l10n";
+import { formatDate } from "../../utils/formatDate";
+import { HibpLikeDbBreach } from "../../utils/hibp";
+import { getLocale } from "../../app/functions/universal/getLocale";
+
+export type Props = {
+  breach: HibpLikeDbBreach;
+  l10n: ExtendedReactLocalization;
+};
+
+export const BreachCard = (props: Props) => {
+  const listFormatter = new Intl.ListFormat(getLocale(props.l10n));
+
+  return (
+    <mj-wrapper border="1px solid #eee" text-align="center" padding="0">
+      <mj-section background-color="#eee">
+        <mj-column vertical-align="middle">
+          <mj-text
+            font-size="18px"
+            line-height="24px"
+            align="center"
+            height="32px"
+          >
+            <BreachLogo breach={props.breach} />
+            <span style={{ paddingInlineStart: "4px" }}>
+              {props.breach.Title}
+            </span>
+          </mj-text>
+        </mj-column>
+      </mj-section>
+      <mj-section padding="20px">
+        <mj-column>
+          <mj-text align="center" font-size="16px" padding="24px">
+            <p style={{ color: "#5e5e72" }}>
```

Comment:
I moved this into its own component because the breach alert email and the signup report email shared it. The only change is that I added the `style` attribute here and on the `<img>` in `<BreachLogo>`, instead of adding a class and a stylesheet to the head - because the component doesn't have access to the head.